service/application/src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ spring:
       enabled: true
       settings:
         web-allow-others: true
+  servlet:
+    multipart:
+      max-request-size: 5MB
+      max-file-size: 5MB
```

Comment:
Would you create a sibling PR on ep-core side to make sure this is generated when creating configs from EP side?

Reply:
yeah, that is a separate task and PR. It is already in progress.